.circleci/config.yml

```diff
@@ -29,7 +29,26 @@ jobs:
           # don't limit this to the number of allocated cores, the job is
           # likely to get OOMed and killed.
           GOOPTS: "-p 2"
-  publish:
+  publish_infra:
+    executor: golang
+    steps:
+    - checkout
+    - setup_remote_docker:
+        version: 18.06.0-ce
```

Comment:
set version to "19.03.8", because of the [update on base image](https://github.com/circleci/circleci-images/issues/487)